modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-migration-process_{context}"]
+= Migrating from oc-mirror plugin v1 to oc-mirror plugin v2
+
+Migrating from oc-mirror v1 to v2 requires manually updating the ImageSetConfiguration, adjusting mirroring commands, and ensuring proper cleanup of v1 artifacts. Follow these steps to transition smoothly:
+
+. Modify the API version and remove deprecated fields in your `ImageSetConfiguration`. Refer below for the examples before and after the configuration:
++
+.Example output with oc-mirror plugin v1 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+storageConfig:  # REMOVE this field in v2
+  local:
+    path: /var/lib/oc-mirror
+----
++
+.Example output with oc-mirror plugin v2 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+----
+
+. Commands have changed to align with oc-mirror plugin v2's new workflow.
+
+.. For mirror-to-disk, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml file://some-dir --v2
+----
+
+.. For disk-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://some-dir docker://remote-registry --v2
+----
+
+.. For mirror-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --workspace file://some-dir docker://remote-registry --v2
+----
++
+[NOTE]
+====
+`--workspace` is now required for mirror-to-mirror operation.
+====
+
+. In v2, mirrored content is stored in different locations. You must check the cluster-resources directory inside their working directory for IDMS, ITMS, `CatalogSource`, and `ClusterCatalog` using the following command :
++
+[source,terminal]
+----
+$ ls <workspace>/cluster-resources/
```

Comment:
I'm not sure this is extra clear for users.
Suggestion: in the previous command line examples, we are using `file://some-dir` .
In the `Mirroring images for a disconnected installation by using the oc-mirror plugin v2`, we are using `file://<file_path>`.
Maybe we should identify this as `file://<v2_workspace` (or any other suggestion: v2_work_path, v2_workspace_path).
This is a way to suggest to the user this is the same folder for m2d, d2m and m2m.
This way, we could say clearly that the yaml files  are in 
```suggestion
$ ls <v2_workspace>/working-dir/cluster-resources/
```